src/H5C.c

```diff
@@ -6579,8 +6579,10 @@ H5C__make_space_in_cache(H5F_t *f, size_t space_needed, hbool_t write_permitted)
     H5C_cache_entry_t *entry_ptr;
     H5C_cache_entry_t *prev_ptr;
     H5C_cache_entry_t *next_ptr;
-    uint32_t           num_corked_entries = 0;
-    herr_t             ret_value          = SUCCEED; /* Return value */
+#if !defined(SYCL_LANGUAGE_VERSION) || !defined(__INTEL_LLVM_COMPILER)
```

Comment:
I don't understand how this can be undefined - it is incremented in line 6631. And the value is used in an assert at line 6758.

Reply:
Ask the developer who wrote h5C.c.
The responsible coder should understand this behavior.